docs/maps/connect-to-ems.asciidoc

```diff
@@ -13,7 +13,16 @@ EMS requests are made to the following domains:
 * vector.maps.elastic.co
 
 **Elastic Maps** makes requests directly from the browser to EMS.
-To proxy EMS requests through the Kibana server, set `map.proxyElasticMapsServiceInMaps` to `true` in your <<settings, kibana.yml>> file.
+
+[float]
+=== Connecting to Elastic Maps Service from an internal network
+
+To connect to EMS when your Kibana server and browser are in an internal network.
```

Comment:
This is a sentence fragment.  Either replace the the ending period with a colon or rewrite as a sentence.

You can connect to EMS  when your Kibana server and browser are in an internal network.